css/projectsModal-style.css

```diff
@@ -0,0 +1,178 @@
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  z-index: 2;
+}
+
+.projectPopup {
+  background: #fff;
+  width: 95%;
+  height: 95%;
+  padding: 1rem 2rem;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.popupHeader {
+  position: relative;
+}
+
+.popupTitle {
+  width: 90%;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  padding-bottom: 0.75rem;
+}
+
+.popupLanguages {
+  display: flex;
+  padding-bottom: 0.75rem;
+}
+
+.popupLanguages li {
+  background-color: #fff;
+  border: 2px solid #8993a4;
+  color: #091e42;
+  padding: 0.25rem 0.75rem;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-weight: 400;
+  font-size: 1rem;
+  margin-right: 0.55rem;
+}
+
+.popupBody {
+  width: 100%;
+}
```

Comment:
- [ ]  As you see below, there is an extra white space at the bottom. This alteration seems to have occurred after your new commits. Kindly fix it.
![image](https://user-images.githubusercontent.com/91486791/232883623-7ac3ee60-3dd4-4364-9f32-457dda4985c1.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/232884109-8aecfff0-b5e5-4199-9fa7-207e242ddd9d.png)

- [ ] Also, as the previous reviewer asked, please check your image path because I can't see them on my browser. (Kindly remove the extra dot before the source path '../images' --> './images')